support/scheduler/router.go

```diff
@@ -0,0 +1,207 @@
+//
+// Copyright (c) 2018 Tencent
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package scheduler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+	mux "github.com/gorilla/mux"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+func LoadRestRoutes() http.Handler {
+	r := mux.NewRouter()
+	b := r.PathPrefix("/api/v1").Subrouter()
+
+	//ping
+	b.HandleFunc("/ping", ping).Methods(http.MethodGet)
+
+	// callback
+	b.HandleFunc("/callbacks", addCallbackAlert).Methods(http.MethodPost)
+	b.HandleFunc("/callbacks", updateCallbackAlert).Methods(http.MethodPut)
+	b.HandleFunc("/callbacks", removeCallbackAlert).Methods(http.MethodDelete)
+
+	return r
+}
+
+const (
+	ContentTypeKey       = "Content-Type"
+	ContentTypeJsonValue = "application/json; charset=utf-8"
+	ContentLengthKey     = "Content-Length"
+)
+
+func ping(rw http.ResponseWriter, req *http.Request) {
+	rw.Header().Set(ContentTypeKey, ContentTypeJsonValue)
+	rw.WriteHeader(http.StatusOK)
+	str := `{"value" : "pong"}`
+	io.WriteString(rw, str)
```

Comment:
Can we use just w.Write instead of io.WriteString? Also seems ping handler is common for all services. is it makes sense move it to some pkg dir?

Reply:
@falkerson thanks for your review, this PR is hosted in [here](https://github.com/edgexfoundry-holding/support-scheduler-go), I would consider your suggestion soon.